src/dotnes.tasks/Utilities/Transpiler.cs

```diff
@@ -139,10 +139,15 @@ public void Write(Stream stream)
         // Pad 0s
         int PRG_ROM_SIZE = (int)writer.Length - 16;
         writer.WriteZeroes(NESWriter.PRG_ROM_BLOCK_SIZE - (PRG_ROM_SIZE % NESWriter.PRG_ROM_BLOCK_SIZE));
-        writer.WriteZeroes(NESWriter.PRG_ROM_BLOCK_SIZE - 6);
 
-        //TODO: no idea what these are???
-        writer.Write([0xBC, 0x80, 0x00, 0x80, 0x02, 0x82]);
+        // Write interrupt vectors
+        const int VECTOR_ADDRESSES_SIZE = 6;
+        writer.WriteZeroes(NESWriter.PRG_ROM_BLOCK_SIZE - VECTOR_ADDRESSES_SIZE);
+        ushort nmi_data = 0x80BC;
```

Comment:
Funnily enough, describing them in high-endianness (what the actual address values are) will result in .NET writing them out in low-endianness (in binary format NES would read). Resulting binary file is still:

`0xBC, 0x80, 0x00, 0x80, 0x02, 0x82`

Reply:
This isn't necessarily related to this PR specifically, but I saw this in the recent Build talk, and now here, and I had to comment.
Most systems, including Windows (x86 and x64) are little-endian. Very few systems are big-endian. We write the values as if it was left to right, big to small, as `int number = 0xaabbccdd;` but the value is always stored least-significant-bit first as `dd cc bb aa`. The `byte` value of "one" (1) is written in code as `0b00000001` but is stored and read in memory as `10000000`

Reply:
I don’t think people comprehend how stressful it is up there. I said it backwards, and I think at least 20 people have already mentioned this. 👀

Reply:
I'm sorry for being a jerk. This project is really great and I enjoyed the talk.

Reply:
No worries!